modules/references/client/components/read-references/Meta.js

```diff
@@ -0,0 +1,42 @@
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+
+const Labels = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 10px 0;
+  .label {
+    margin-right: 5px;
+  }
+`;
+
+export default function Meta({ met, hostedMe, hostedThem, recommend }) {
+  const { t } = useTranslation('references');
+
+  return (
+    <Labels>
+      {[
+        recommend === 'yes' && t('Recommend'),
+        recommend === 'no' && t('Not recommend'),
+        hostedMe && t('Guest'),
+        hostedThem && t('Host'),
+        met && t('Met in person'),
+      ]
+        .filter(label => !!label)
+        .map(label => (
+          <span key={label} className="label label-default">
+            {label}
+          </span>
```

Comment:
is this case too extreme? :D

<image src="https://user-images.githubusercontent.com/2955794/101995492-717cd980-3cca-11eb-806d-527f0a3cb640.png" width="300px"/>

Reply:
Haha, not at all. :-D I'm actually not a big fan of just a list of labels like this, but it was a quick solution. I'm still thinking about how to present them better. Any ideas welcome!

Meanwhile, I can make sure it doesn't break on mobile.

Mobile navigations clearly need some work, too.

Reply:
Aight, fixed in https://github.com/Trustroots/trustroots/pull/1902/commits/2c9c8038398f6a7e7d88ed45bfe4588dd6d47fb0

![image](https://user-images.githubusercontent.com/87168/102013967-0845a680-3d5c-11eb-96dc-a7f4a5c7383c.png)